styles/internal/fonts.less

```diff
@@ -0,0 +1,68 @@
+// ----- SYSTEM FONT NAMES ------
+@font-system-src-moonstone-icons:  local("Moonstone-MyTheme"), url("../../fonts/Moonstone-MyTheme.ttf") format("truetype");
```

Comment:
Should we illustrate a non-moonstone (maybe another google font like Material Fonts) usage here?

Reply:
Implemented Material Icons